requirements.txt

```diff
@@ -3,3 +3,4 @@ pyodbc>=4.0.30
 sqlparams>=3.0.0
 thrift>=0.13.0
 sqlparse>=0.4.2 # not directly required, pinned by Snyk to avoid a vulnerability
+requests>=2.28.0 # needed for spark-cde
```

Comment:
Adding to requirements as we need this library to make http requests.

Reply:
we need the following as well:
requests-toolbelt>=0.9.1
python-decouple>=3.6

I though this was added in an earlier PR.